docs/docusaurus.config.js

```diff
@@ -176,6 +176,9 @@ module.exports = {
       apiKey: "124fe1c102a184bc6fc70c75dc84f96f",
       indexName: "zitadel",
       selector: "div#",
+      getMissingResultsUrl({ query }) {
+        return `https://github.com/zitadel/zitadel/discussions/new?category=ideas&title=Missing docs for ${query}`;
+      },
```

Comment:
```suggestion
```

Reply:
lets leave this aside and fix the deps first (since this is not working)

Reply:
done